src/main/kotlin/uk/gov/justice/digital/hmpps/subjectaccessrequestworker/client/ProbationApiClient.kt

```diff
@@ -61,4 +73,25 @@ class ProbationApiClient(
       )
     }
   }
+
+  @JsonIgnoreProperties(ignoreUnknown = true)
+  data class GetOffenderDetailsResponse(
+    @JsonProperty("name")
+    val nameDetails: NameDetails?,
+  )
+
+  data class NameDetails(val surname: String? = "", val forename: String? = "") {
+    /**
+     * Return the offender name formatted as "SURNAME, forename" if both values present. Otherwise, return empty string
+     */
+    fun formatName(): String {
+      if (StringUtils.isEmpty(surname) || StringUtils.isEmpty(forename)) {
+        return ""
+      }
+
+      return "${surname!!.uppercase()}, ${forename!!.lowercase().replaceFirstChar { it.titlecase() }}"
+    }
```

Comment:
was just thinking if should make this a common function as it's the same as in GetOffenderDetailsResponse, or whether not worth it or want to keep the separation?